src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyStatusMapper.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.companieshouse.insolvency.delta.mapper;
+
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.ADMINISTRATION_ORDER;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.ADMINISTRATIVE_RECEIVER;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.IN_ADMINISTRATION;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.LIQUIDATION;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.LIVE_RECEIVER_MANAGER_ON_AT_LEAST_ONE_CHARGE;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.RECEIVERSHIP;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.RECEIVER_MANAGER;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.VOLUNTARY_ARRANGEMENT;
+import static uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum.VOLUNTARY_ARRANGEMENT_RECEIVERSHIP;
+
+import java.util.List;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.insolvency.CompanyInsolvency.StatusEnum;
+
+@Component
+public class InsolvencyStatusMapper {
+
+    /**
+     * @param key the chips key for insolvency status.
+     * @return list of insolvency statuses.
+     */
+    public List<StatusEnum> mapStatus(final String key) {
+        List<StatusEnum> status = null;
+
+        if (StringUtils.isNotBlank(key)) {
```

Comment:
I think this would be better implemented as either a StatusList enum or a lookup HashMap in config